### ISSUE TITLE
icicles.el is compressed in icicles-20141215.1749, causing install error

### ISSUE THREAD

**Issue by makotoy** [milkypostman/melpa#2366]
The file icicles.el in icicles-20141215.1749 is gziped while it should be a plain source file, which causes the package not to install. If you try to upgrade an existing install to this version, or newly install this via packages.el machinery the installation aborts with the error: forward-sexp: Scan error: "Containing expression ends prematurely". I tried gunzipping it and the result seems fine.

**Comment by purcell** [comment]
I believe `icicles` is from the emacswiki, which was having some problems recently and was serving some files with strange encodings.

Right now `curl -s -D- -o /tmp/icicles.el` shows the headers 

```
HTTP/1.1 200 OK
Server: nginx/1.4.6 (Ubuntu)
Date: Wed, 07 Jan 2015 14:24:53 GMT
Content-Type: text/html; charset=utf-8
Content-Length: 23049
Connection: keep-alive
Vary: Accept-Encoding
Content-Encoding: gzip
X-EmacsWiki-Source: Apache/CGI
X-Proxy-Cache: HIT
```

and the output file is gzipped. Same with wget. But neither `curl` nor `wget` send an `Accept-Encoding` header mentioning `gzip`, so I think the content negotiation is going wrong on the emacswiki side. If I do the same thing with a different site which supports gzip, for example, no `Content-Encoding: gzip` header is returned. Hey @nicferrier, what do you think?

**Comment by milkypostman** [comment]
we had this problem before. my guess is that the proxy is not correctly
handling encodings. so it's getting confused when someone requests non-gzip.

**Comment by nicferrier** [comment]
You don't say what url you're pulling.

**Comment by milkypostman** [comment]
whoops, here you go:

http://www.emacswiki.org/emacs/download/icicles.el

**Comment by nicferrier** [comment]
No more gzip.

**Comment by purcell** [comment]
Super, thanks a lot Nic!

**Comment by makotoy** [comment]
Updated to icicles-20150111.1915 without trouble, thanks!

**Comment by nicferrier** [comment]
Apparently we're still having trouble but I'm not seeing the issues except through the occassional github mail which is fractured and hard to follow. So if there's a problem please do let me know.

**Comment by milkypostman** [comment]
going to close. someone can open a new PR when it's needed and i suspect more and more we will be making changes to the way melpa pulls wiki packages.
